lib/utils/logger.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+const util = require('util');
+
+function defaultInspector(value) {
```

Comment:
Get rid of this, just use `util.inspect(value, false, 3);` directly in `inspect` method